CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ if(ENABLE_LLVM)
   add_compile_definitions(USE_LLVM)
 endif()
 cmake_dependent_option(ENABLE_GPU "Enable support for GPUs" ON "NOT ENABLE_LLVM" OFF)
-cmake_dependent_option(ENABLE_COMPILED "Include compiled GPU kernels in comparison" OFF "ENABLE_CUDA" OFF)
+cmake_dependent_option(ENABLE_COMPILED "Include compiled GPU kernels in comparison" OFF "ENABLE_GPU" OFF)
```

Comment:
Can you briefly explain what this line means? Like two `OFF` here?

Reply:
> Thanks @K20shores for working on this integration. How could I run your code on Casper to verify the functionality of Openacc on GPU?

@sjsprecious, I had forgotten about linking the libraries (yay lack of sleep...).

Anyways, to compile this you can run `cmake -DENABLE_LLVM=OFF -DENABLE_GPU=ON -DENABLE_COMPILED=OFF ..` from the build directory. And then run `make` and `make test`. As you will see, right now something about the openacc version fails, probably something that I changed and broke. I'm investigating now

Reply:
`ENABLE_COMPILED` is an option that is only available when `ENABLE_GPU` is set to ON. In this case, the two `OFF` values are specifying that the default value for `ENABLE_COMPILED` is OFF. We included code that would take the string in the program for the JIT cuda kernel and write it to a file so that you could then configure cmake to compile these cuda kernels to compare the performance of JITed cuda kernels to string cuda kernels.

Reply:
`cmake -DENABLE_LLVM=OFF -DENABLE_GPU=ON -DENABLE_COMPILED=OFF ..` is what I recommend you run. I think we would need to do more work to make `-D ENABLE_COMPILED` to work. It tries to use code that is generated in the derivative test.

Reply:
Thanks!